src/presence_analyzer/templates/mean_time_weekday.html

```diff
@@ -60,16 +48,16 @@
             });
         })(jQuery);
     </script>
-</head>
+{% endblock %}
 
-<body>
+{% block body %}
     <div id="main">
         <div id="header">
             <h1>Presence analyzer</h1>
             <ul>
-                <li><a href="/static/presence_weekday.html">Presence by weekday</a></li>
-                <li id="selected"><a href="/static/mean_time_weekday.html">Presence mean time</a></li>
-                <li><a href="/static/presence_start_end.html">Presence start-end</a></li>
+                <li><a href="{{ url_for('mainpage') }}">Presence by weekday</a></li>
+                <li id="selected"><a href="{{ url_for('mean_time_page') }}">Presence mean time</a></li>
+                <li><a href="{{ url_for('presence_start_end_page') }}">Presence start-end</a></li>
```

Comment:
This is repeated, use a loop in which u'll generale `li' and set active page + add it to separate html file and include it It should be included in `base.html`